packages/react/src/components/Storage/FileUploader/UploadTracker/UploadTracker.tsx

```diff
@@ -63,6 +65,12 @@ export function UploadTracker({
     fileState === FileState.INIT ||
     (fileState === FileState.ERROR && errorMessage === extensionNotAllowedText);
 
+  useEffect(() => {
+    if (shouldAutoLoad && fileState === FileState.INIT) {
+      startUpload();
+    }
+  });
```

Comment:
We probably don't want this to run on every render right?

Reply:
yeah, pushed too early